Update User Guide - Unadvertised resources (#9976)
geosolutions-it/MapStore2: docs/user-guide/resources-properties.md

Through the *Edit properties* window the user can perform the following operations:

* Add an image as a **Thumbnail** dropping it or clicking inside the *Thumbnail* box.

!!! warning
    The image to be added must not be larger than 500 kb and its best dimensions are 300x180 px. The supported formats are `jpg` (or `jpeg`) and `png`.

* Add a **Name** and a **Description**

!!! note
    The name of a resource is the only mandatory field. Note that is not allowed to choose a name that has already been assigned to another resource.

> * Enable the **Unadvertised** option to makes the resource owner or the admin the only one who can see and search the resource on the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.

Reviewer: ```suggestion
* Enable the **Unadvertised** option to makes the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```

> * * Enable the **Unadvertised** option to make the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.

Reviewer 2: ```suggestion
* Enable the **Unadvertised** option to make the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```

> * * Enable the **Unadvertised** option to makes the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.

Reviewer: ```suggestion
* * Enable the **Unadvertised** option to make the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```

> * Add a **Permission** rule

Reviewer: Please put a link to the related section.

* Add **Details** (only for *Maps* and *Dashboard*)

> Once a resource is saved, the *Edit properties* panel shows the creation and last modification date. An example in the image below:

Reviewer: ```suggestion
Once a resource is saved, the *Edit properties* panel shows the creation and the last modification dates. An example in the image below:
```

<img src="../img/resource-properties/resource_data.jpg" class="ms-docimage"  style="max-width:400px;"/>

> While an admin user can also view who created the resource and the last user who modified it. An example in the image below:

Reviewer: ```suggestion
Admin users can also see who created and modified the resource. An example in the image below:
```

## Details

> Only for resources of type map and dashbaord, it is possible to add details to the resource. This is useful to associate some information to the resource or an overview description of its content. In this case the *Edit properties* window is the following:

Reviewer: ```suggestion
Only for resources of type _map_ and _dashbaord_, it is possible to add details as additional information in form of descriptive content for the resource itself. This is useful to associate some information to the resource or an overview description of its content. In this case the *Edit properties* window is the following:
```

> Only for resources of type *map* and *dashboard*, it is possible to add details as additional information in form of descriptive content for the resource itself. This is useful to associate some information to the resource or an overview description of its content. In this case the *Edit properties* window is the following:

Reviewer: ```suggestion
Only for resources of type *map* and *dashboard*, it is possible to add details as additional information in form of descriptive content for the resource itself. This is useful to associate some information to the resource and provide an overview of its content. In this case the *Edit properties* window is the following:
```

<img src="../img/resource-properties/edit-map-properties-panel_details.jpg" class="ms-docimage"  style="max-width:400px;"/>

> With a click on the **Add new details** button <img src="../img/button/add_details_button.jpg" class="ms-docbutton"/> it opens a panel where the user can write the details of the resource.

Reviewer: ```suggestion
With a click on the **Add new details** button <img src="../img/button/add_details_button.jpg" class="ms-docbutton"/> a panel opens and the user can write the details of the resource.
```

!!! warning
    The *About this map* button is visible in the [Side Toolbar](mapstore-toolbars.md#side-toolbar) only when the details are present on the map.

> * Enable the **Show at startup** <img src="../img/button/show-at-starup-button.jpg" class="ms-docbutton"/> button. If active, as soon as the user opens the resources, the details panel is visualized.

Reviewer: ```suggestion
* Enable the **Show at startup** <img src="../img/button/show-at-starup-button.jpg" class="ms-docbutton"/> button. If active, when the user opens a resources with details defined, these are visualized in a descriptive panel.
```
